modules/oncodrivefml/1.0/config/default.yaml

```diff
@@ -0,0 +1,70 @@
+lcr-modules:
+    
+    oncodrivefml:
+        
+        inputs:
+            # Available wildcards: {seq_type} {genome_build}
+            input_maf: "__UPDATE__"
+            subsetting_categories: "__UPDATE__"
+
+        genome_builds: ["__UPDATE__"]
+        # Available regions options: cds, lincrna, 3utr, 5utr, promoter_splice_sites
+        regions: ["__UPDATE__"]
+
+        maf_processing:
+            sample_sets: ["__UPDATE__"]
+            prepare_mafs: "{SCRIPTSDIR}/generate_smg_inputs/1.1/generate_smg_inputs.R"
+            include_non_coding: False
+            blacklists: []
+            blacklist_drop_threshold: 4
+
+        format_fml_input:
+            input_columns: "5,6,11,13,16"
```

Comment:
can you please add a comment here on what are these numbers?

Reply:
Added more details:
        format_fml_input:
            # String in input_columns is passed to the cut command to subset MAF columns to only include:
            # Chromosome, Start_Position, Reference_Allele, Tumor_Seq_Allele2, Tumor_Sample_Barcode
            # e.g "cut -f 5,6,11,13,16"
            input_columns: "5,6,11,13,16"
            # If additional columns are incorporated, add additional commands here to change column names if necessary
            # e.g additional_commands: "sed 's/Normal_Sample_Barcode/SIGNATURE/'"
            # More info: https://oncodrivefml.readthedocs.io/en/latest/files.html#file-formats
            additional_commands: ""

Reply:
Added more info:
>       format_fml_input:
            # String in input_columns is passed to the cut command to subset MAF columns to only include:
            # Chromosome, Start_Position, Reference_Allele, Tumor_Seq_Allele2, Tumor_Sample_Barcode
            # e.g "cut -f 5,6,11,13,16"
            input_columns: "5,6,11,13,16"
            # If additional columns are incorporated, add additional commands here to change column names if necessary
            # e.g additional_commands: "sed 's/Normal_Sample_Barcode/SIGNATURE/'"
            # More info: https://oncodrivefml.readthedocs.io/en/latest/files.html#file-formats
            additional_commands: ""